torch/onnx/symbolic_opset11.py

```diff
@@ -375,7 +375,7 @@ def symbolic_fn(g, input, kernel_size, stride, padding, ceil_mode, count_include
 
 
 @parse_args("v", "i", "i", "i", "i")
-def unique_dim(g, self, dim, sorted, return_inverse, return_counts):
+def _unique_dim(g, self, dim, sorted, return_inverse, return_counts):
```

Comment:
Hmm, this change scares me a little. If anyone used unique_dim in an onnx graph then we are breaking them. @BowenBao (since you work with ONNX), is that a correct analysis?